src/main/scala/org/globalnames/parser/ParserClean.scala

```diff
@@ -310,8 +310,8 @@ class ParserClean extends SimpleParser {
 
   val unknownAuthor: Rule1[Author] = rule {
     capture("?" |
-            (("auct." | "auct" | "anon." | "anon" | "ht." | "ht" | "hort." |
-              "hort") ~ &(spaceChars | EOI))) ~>
+            (("auct" | "anon" | "ht" | "hort") ~ '.'.? ~
```

Comment:
Do we need to capture exact string of anonymous author? In other words, should anonymous author be normalized to, say, `"anon."`, or exact string from verbatim should be preserved?